docs/modules/intellij-idea-plugin/pages/user-guide.adoc

```diff
@@ -0,0 +1,82 @@
+= Payara IntelliJ IDEA Plugin User Guide
+:man-prefix: #
+:man-suffix: 1
+:imagesdir: images/
+
+== Introduction
+
+The Payara Ultimate Tools plugin enables integration of Payara Cloud and Platform with IntelliJ IDEA Ultimate Edition. The Payara Tools plugin is available in the https://plugins.jetbrains.com/plugin/15114-payara-platform-tools[JetBrains Plugins Marketplace].
+
+If you are developing an application on IntelliJ IDEA Community Edition, you should install the Payara Community Tools plugin, which enables integration of Payara Cloud and Payara Micro with IntelliJ IDEA Community Edition. The Payara Community Tools Plugin is also available in the https://plugins.jetbrains.com/plugin/15445-payara-micro-community-tools[JetBrains Plugins Marketplace].
```

Comment:
OK I Payara Community Tools Plugin could suggest it be used for Payara Community.